src/js/binaryen.js-post.js

```diff
@@ -0,0 +1,29 @@
+return Module;
```

Comment:
Instead of the `pre` and `post` bits to wrap and return the module, you can use emcc's `MODULARIZE` option. although then the code outside the function must be added after `emcc`, so maybe not worth it.